cukinia

```diff
@@ -524,4 +610,12 @@ cukinia_tests=0
 cukinia_failures=0
 cukinia_conf_include "$CUKINIA_CONF"
 
+# Create junitxml report
+if [ "$__log_format" = "junitxml" ]; then
+	_junitxml_start >"$__log_file".xml
```

Comment:
use a tmp file during generation

Reply:
we must have an output file for junit so why not use this sequence to create it? it is the last moment where we can do it. Moreover it is a parameter, so it incombs to the user to provide a writable file for its output. We can put a guard though.

Reply:
if you Ctrl-C during the test suite, the xml file will be left truncated incomplete. From here I see two solutions:

* either use a temporary file name and once the file is complete, mv tmpfile thefile.xml
* make cleanup() more complicated to finalize the xml in case. I don't like it because for example if you break at test 3 of 10 and tests 1..3 PASS well you will have an xml with everything good.

Reply:
Ack done